src/components/EscapeRoom/Room.jsx

```diff
@@ -560,7 +538,7 @@ export default function Room({ handleEscape, setIsRoomLoaded })
                 setExitDoor(true)
                 break;
             case intersectsArray.includes('Exit_Screen') && exitDoor:
-                handleEscape();
+                console.log("You Escaped!");
```

Comment:
This needs to be changed back to handleEscape()

Reply:
Also need to change line 12 back to export default function Room({ handleEscape, setIsRoomLoaded })

![image](https://github.com/CS467-AI-Coder-Escape-Room/CS467-AI-Coder-Escape-Room/assets/32920594/7bac05b4-657a-471d-87a0-43fd159fa04c)